crates/bevy_math/src/cubic_splines.rs

```diff
@@ -308,12 +358,37 @@ impl<P: VectorSpace> CubicGenerator<P> for CubicBSpline<P> {
             .iter_mut()
             .for_each(|r| r.iter_mut().for_each(|c| *c /= 6.0));
 
+        char_matrix
+    }
+}
+impl<P: VectorSpace> CubicGenerator<P> for CubicBSpline<P> {
+    #[inline]
+    fn to_curve(&self) -> CubicCurve<P> {
         let segments = self
             .control_points
             .windows(4)
-            .map(|p| CubicSegment::coefficients([p[0], p[1], p[2], p[3]], char_matrix))
+            .map(|p| CubicSegment::coefficients([p[0], p[1], p[2], p[3]], self.char_matrix()))
+            .collect();
+
+        CubicCurve { segments }
+    }
+}
+
+impl<P: VectorSpace> CyclicCubicGenerator<P> for CubicBSpline<P> {
+    #[inline]
+    fn to_curve_cyclic(&self) -> CubicCurve<P> {
+        let segments = self
+            .control_points
+            .iter()
+            .circular_tuple_windows()
+            .map(|(&a, &b, &c, &d)| CubicSegment::coefficients([a, b, c, d], self.char_matrix()))
             .collect();
 
+        // Note that the parametrization is consistent with the one for `to_curve` but with
+        // the extra curve segments all tacked on at the end. This might be slightly counter-intuitive,
+        // since it means the first segment doesn't go "between" the first two control points, but
+        // between the second and third instead.
+
```

Comment:
I think these things would be good to have in the documentation. (Applies to all curves.)

Reply:
I ended up adding a "Parametrization" section to the docs to describe these things. Let me know what you think!